src/config/development.json

```diff
@@ -1,4 +1,5 @@
 {
+	"css_url": "http://localhost:9000/happychat.css",
```

Comment:
Maybe we could store the `localhost:9000` url in a separate field as `root`? Though, it can't be reused within the same file..

Reply:
I don't think we need to worry about that right now.

Reply:
I like this approach better!